core/meta/user/users.ts

```diff
@@ -0,0 +1,8 @@
+import { User } from './user'
+import { RoleName, UserRole } from './userRole'
+
+const isAdministrator = (user: User) => user.roles.filter((role: UserRole<any>) => role.role === RoleName.ADMINISTRATOR)
```

Comment:
this should return a boolean, not an array of roles.
use Boolean(user.roles.find)

Reply:
what do you think about this? https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/some

Reply:
no because there's only one role administrator + cast the result to boolean